bugaenko/README.md

```diff
@@ -0,0 +1,114 @@
+```bash
+
+1  
+
+149  cd /home/ss/git/02.git/
+  150  ls
+  151  cler
+  152  clear
+  153  git init
+  154  nano README.md
+  155  ls
+  156  git status
+  157  git commit -m "empty"
+  158  git add --all
+  159  git status
+  160  git commit
+  161  git status
+2
+ 
+ 162  git branch feature-branch
```

Comment:
was this brach created before? looks like you switch to this brach without create